docs/getting_started/the_zen_of_wagtail.md

```diff
@@ -0,0 +1,33 @@
+# The Zen of Wagtail
+
+Wagtail has been born out of many years of experience building websites, learning approaches that work and ones that don\'t, and striking a balance between power and simplicity, structure and flexibility. We hope you\'ll find that Wagtail is in that sweet spot. However, as a piece of software, Wagtail can only take that mission so far - it\'s now up to you to create a site that\'s beautiful and a joy to work with. So, while it\'s tempting to rush ahead and start building, it\'s worth taking a moment to understand the design principles that Wagtail is built on.
```

Comment:
Not sure why we need to escape quotes in markdown?

Reply:
I don't think we do. This is just how it comes out of Pandoc... I can remove those.

Reply:
Thanks, this PR looks good to me otherwise!

Reply:
Fixed in eb1a17cfa62e1bd34b5b6535ea09b4483f65e5ef